src/modules/appeals-service/AppealsService.ts

```diff
@@ -37,6 +42,38 @@ export class AppealsService {
 
     }
 
+    public async hasExistingAppeal(companyNumber: string, penaltyReference: string, accessToken: string,
```

Comment:
It would look nicer if you stacked the arguments vertically

Reply:
tslint: Exceeds maximum line length of 120 (max-line-length)